tests/runtime/symlinks/test.wake

```diff
@@ -6,12 +6,14 @@ export def goodLink Unit =
         datFile Unit
     makeExecPlan ("ln", "-s", "datFile", "goodLink", Nil) (dat, Nil)
     | setPlanLabel "test (symlinks): goodLink"
+    | setPlanKeep False
```

Comment:
I'm pretty sure we don't *want* tests to be cached -- at best nothing happens, but it seems to me that doing so risks returning stale data.  Since by its nature it's calling out to a shell command rather than running Wake code itself I guess it *should* be safe, but it seems to me that ensuring every test invocation reruns everything from scratch better matches the model of "does this still do what we want, after any changes we may have made".

Reply:
Except where otherwise desired these tests also test interaction with the database.

Reply:
Ah, I see where that's occurring.  I'm not sure that's great behaviour since it's not particularly obvious how we'd indicate a test failure occurring due to an error in Wake vs. due to an error in the database beyond hoping people notice whether the error occurred on the first or the second time through any given test stage (and I generally prefer more atomic test definitions than anything we've set up currently), but since we do at least remove the `wake.db` in the runners I'll raise these to just avoid sharing.